package.json

```diff
@@ -84,14 +90,15 @@
     "lint": "npm-run-all lint:*",
     "lint:js": "eslint src",
     "lint:js:fix": "eslint src --fix",
-    "start": "fec dev",
     "start:stage": "fec dev --clouddotEnv=stage",
     "start:prod": "fec dev --clouddotEnv=prod",
     "start:msw:stage": "NODE_ENV=development MSW=TRUE fec dev --clouddotEnv=stage",
     "test": "TZ=UTC vitest run",
     "test:watch": "TZ=UTC vitest",
     "test:coverage": "TZ=UTC vitest run --coverage",
     "build": "fec build",
+    "cockpit:build": "webpack --config cockpit/webpack.config.ts && npm run cockpit:buildln",
+    "cockpit:buildln": "ln -s $(pwd)/cockpit/public ~/.local/share/cockpit/image-builder-frontend",
```

Comment:
nitpick: the symlink can be added in a first setup, i.e `cockpit:setup`.  it's not needed for every build.

Reply:
is it really metter if its build or setup definition?

Reply:
setup is for initial/one-time tasks like the host patch for adding consoleDot DNS records. Building the cockpit bundle is a repetitive task, which also will be triggered from CI automation, so I think it would better to keep the logic separated.  Keep in mind that with no webpack dev server, a build is required for every change during development.

Reply:
Do we want to do this all in `package.json`? The Cockpit project has a "starter kit" repo (https://github.com/cockpit-project/starter-kit/) which uses a Makefile to set up the development environment - would it make sense to use a similar pattern here? 

See here:
https://github.com/cockpit-project/starter-kit/blob/4d8fc78a3f32e0c7b7abc6961c2f54e8ccd47514/Makefile#L105

Reply:
Creating a makefile just for cockpit setup?
Also, notice that the starter-kit switched to esbuild from webpack.

Reply:
I think that it is make sense to leave it at package.json,
all the scripts in one place- we build Saas env there and also build the onPrem env